docs/deploying/BBSSignatures.md

```diff
@@ -0,0 +1,27 @@
+# BBS+ Signatures Support
+
+ACA-Py includes support for BBS+ Signatures out of the box through an optional dependency, however the implementation for it is out of date and does not currently support ARM (aarch64 or arm64) architecture (e.g. Apple devices with M1 chips or above).
```

Comment:
How about updating this paragraph to the following to better describe what we are doing:

> ACA-Py has supported BBS Signatures for some time. However, the dependency that is used does not support the ARM architecture, and its inclusion in the default ACA-Py artifacts mean that developers using MacOS M1 hardware cannot run ACA-Py “out-of-the-box”.  We feel that providing a better developer experience by supporting M1 hardware is more important than BBS Signature support at this time.  As such, we have removed the BBS dependency from the base ACA-Py artifacts, and made it an add on, that those using ACA-Py with BBS must take extra steps to build their own artifacts. This file describes how to do those extra steps.
>
> Regarding future support for BBS Signatures in ACA-Py. There is currently a lot of work going on in developing BBS Signature implementations and BBS-based verifiable credential standards. However, at the time of this release, there is not an obvious approach and BBS Signatures implementation to use in ACA-Py that includes M1 ARM support. As a result, we will hold off on updating the BBS Signatures support in ACA-Py until the standards and path forward clarify. In the meantime, the Maintainers of ACA-Py plan to continue to do all we can to push for newer and better ZKP-based verifiable credential standards.